plugins/generator-1.18.2/forge-1.18.2/utils/triggers.java.ftl

```diff
@@ -197,6 +197,28 @@
 </#if>
 </#macro>
 
+<#macro hasGlow procedure="">
+@Override @OnlyIn(Dist.CLIENT) public boolean isFoil(ItemStack itemstack) {
+   	<#if hasProcedure(procedure)>
+    <#assign dependencies = procedure.getDependencies(generator.getWorkspace())>
+    <#if !(dependencies.isEmpty() || dependencies.size() == 1 && dependencies.get(0).getName() == "itemstack")>

```

Comment:
I would wrap || and && in brackets just to make it more clear on the operator order

Reply:
Also, does this work if `dependencies.isEmpty()` as then `dependencies.get(0)` is null?

Reply:
If `dependencies.isEmpty()`, then it doesn't check the other conditions because it's already true

Reply:
Ok, that's why I think adding parentheses would help to make this more clear